lib/project_types/theme/messages/messages.rb

```diff
@@ -31,6 +32,14 @@ module Messages
             APP
           },
         },
+        serve: {
+          help: <<~HELP,
+            Open your theme in your browser and print the url in your terminal to view changes in real time.
```

Comment:
Feels like this glosses over exactly what's happening, I think we can be a little more detailed with the explanation — this is the extended help, after all!

Just to clarify what occurs when the user runs `theme serve`:
- Theme Kit syncs the local theme files to the remote ones on the store (it's an upstream push, correct?) 
- The CLI outputs the URL of the store (whether production or dev store, I assume it doesn't matter?)
  - Plus any other terminal messages they'd see in the console at this point?
- The CLI also opens the store homepage in the system's default browser
- At this point they can make local changes and those will continue to be pushed upstream? (like, there's no need to pass a `--watch` option or anything)

Reply:
Yes, that's all correct. The terminal messages following the link would be updates on their changes as they are being pushed upstream.

Reply:
Cool, let's try to capture a little more of it in that case:

```suggestion
            Sync your current theme changes, then view the active store in your default browser. Any theme edits will continue to update in real time. Also prints the active store’s URL in your terminal.
```